### DISCUSSION TITLE
Add support for cycles other than 1/3 1/2 2/3.

### DISCUSSION THREAD

**Discussion by rovrevik** [rxhanson/Rectangle#452] [Ideas]
With larger screens I would be helpful to have cycles other than the fixed 1/3, 1/2, 2/3.
Four configurable cycle properties would be ideal.
Maybe adding a 3/4 would be sufficient.

**Comment by rxhanson** [comment]
I don't have plans to add this into Rectangle at this time, although if someone wanted to create a pull request with an elegant way to do it then I would consider it.

This is a capability of the custom shortcut creator in my paid superset of Rectangle, [Hooskhot](https://hookshot.app). Here's a post on how to use it: https://medium.com/ryan-hanson/custom-window-dimension-shortcuts-in-macos-e7df347dfd24
